src/main/scala/org/broadinstitute/dsde/firecloud/service/ExportEntitiesByTypeActor.scala

```diff
@@ -119,56 +129,64 @@ class ExportEntitiesByTypeActor(rawlsDAO: RawlsDAO,
    */
 
   // Standard exceptions have to be handled as a completed request
-  private def handleStandardException(t: Throwable): Future[Unit] = {
+  private def handleStandardException(t: Throwable): Future[HttpResponse] = {
     val errorReport = t match {
       case f: FireCloudExceptionWithErrorReport => f.errorReport
       case _ => ErrorReport(StatusCodes.InternalServerError, s"FireCloudException: Error generating entity download: ${t.getMessage}")
     }
-    Future(ctx.complete(HttpResponse(
-      status = optAkka2sprayStatus(errorReport.statusCode).getOrElse(StatusCodes.InternalServerError),
-      entity = HttpEntity(ContentTypes.`application/json`, errorReport.toJson.compactPrint))))
+    Future(HttpResponse(
+      status = errorReport.statusCode.getOrElse(StatusCodes.InternalServerError),
+      entity = HttpEntity(ContentTypes.`application/json`, errorReport.toJson.compactPrint)))
   }
 
-  // Stream exceptions have to be handled by directly closing out the RequestContext responder stream
-  private def handleStreamException(t: Throwable): Unit = {
-    logger.info("handling exception", t)
-    val message = t match {
-      case f: FireCloudExceptionWithErrorReport => s"FireCloudException: Error generating entity download: ${f.errorReport.message}"
-      case _ => s"FireCloudException: Error generating entity download: ${t.getMessage}"
-    }
-    ctx.responder ! MessageChunk(message)
-    ctx.responder ! ChunkedMessageEnd
-  }
+  private def streamSingularType(entityQueries: Seq[EntityQuery], metadata: EntityTypeMetadata, entityHeaders: IndexedSeq[String]): Future[File] = {
+    val tempEntityFile: File = File.newTemporaryFile(prefix = entityType)
+    val entitySink: Sink[ByteString, Future[IOResult]] = FileIO.toPath(tempEntityFile.path)
```

Comment:
do you have any idea when (if?) these temp files are cleaned up?

Reply:
they're in the container at `/tmp` (just verified on dev), so they'd get destroyed any time we redeploy but i suspect they'd also get cleaned up at least once a month by the OS